flask_app/app.py

```diff
@@ -139,34 +166,52 @@ def rest_edit(id):
             rest_to_edit.phone = request.form.get('phone') 
             db.session.commit()
             return redirect(url_for('rest'))
-        return render_template('rest_edit.html', restaurant=rest_to_edit)
+        return render_template('rest_edit.html', restaurant=rest_to_edit,current_user=current_user)
     else:
         flash('Restaurant not found')
     return redirect(url_for('rest'))
 
-@app.route('/register', methods=['POST', 'GET'])
+
+@app.route('/register', methods=['GET', 'POST'])
 def register():
     if request.method == 'POST':
-        print(request.form)  # This will print the form data to the console
-        name = request.form.get('name')  
+        print(request.form)
+        name = request.form.get('name')
         email = request.form.get('email')
         phone = request.form.get('phone')
+
         plain_text_password = request.form.get('password')
 
-        if not plain_text_password: 
+        if not plain_text_password:
             return "Password is required", 800
 
         hashed_password = generate_password_hash(plain_text_password)
-        
-        new_user = User(name=name, email=email, phone=phone, password=hashed_password)
-        db.session.add(new_user)
-        db.session.commit()
-        
+        if request.form.get('email') == 'admin@admin.admin':
```

Comment:
Why is it hard coded? Can't admin have a different email than admin@admin.admin ?

Reply:
it is hard coded bcs of issue when zero users u cant add admin permisions in edit user bcs you dont have admin when register

Admin can add another admin email learn to read commit message: 

"Edit users can add Admin"